ingress-inquisition/utils/awsutils.go

```diff
@@ -2,18 +2,22 @@ package utils
 
 import (
 	"context"
+	"fmt"
+	"os"
+	"text/tabwriter"
 
 	"github.com/aws/aws-sdk-go-v2/service/ec2"
 	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	"github.com/aws/aws-sdk-go-v2/service/securityhub"
 	"github.com/guardian/fsbp-tools/common"
 )
 
-type VpcDetails struct {
```

Comment:
Lower casing structs means they are no longer exported. This is fine, as these are only exposed publicly as part of the larger `SecurityGroupRuleDetails` struct